meta-balena-common/recipes-containers/balena/balena/balena.service

```diff
@@ -10,8 +10,7 @@ Type=notify
 Restart=always
 SyslogIdentifier=balenad
 ExecStart=/usr/bin/healthdog --healthcheck=/usr/lib/balena/balena-healthcheck /usr/bin/balenad --experimental --log-driver=journald -s @BALENA_STORAGE@ -H fd:// -H unix:///var/run/balena.sock -H unix:///var/run/balena-engine.sock --dns 10.114.102.1 --bip 10.114.101.1/24 --fixed-cidr=10.114.101.0/25 --max-download-attempts=10 --exec-opt native.cgroupdriver=systemd
-ExecStartPost=/bin/sleep 15
-ExecStartPost=/usr/bin/balena-engine load -i /usr/lib/balena/hello-world.tar
+ExecStartPost=/bin/bash -c '/usr/lib/balena/balena-healthcheck-image-load &'
```

Comment:
I see in the commit log that unless executed in the background the service gets stuck. However, I don't understand why this is needed. ExecStartPost should execute balena-engine-load and the command should exit with success, flagging the service as running.
Do you have an explanation?